core/src/test/kotlin/org/stellar/anchor/auth/AuthHelperTest.kt

```diff
@@ -69,6 +65,8 @@ class AuthHelperTest {
         val wantCustodyAuthHeader =
           AuthHeader("Authorization", "Bearer ${jwtService.encode(wantCustodyJwt)}")
         assertEquals(wantCustodyAuthHeader, gotCustodyAuthHeader)
+        unmockkAll()
+        unmockkStatic(Calendar::class)
```

Comment:
I thought `unmockkStatic` and `mockkAll` were problematic?

Reply:
The unmockStatic only unmock one class. It is better than `clearAll()`. When we removed `mockkStatic(Calender::class)` in `@SetUp`, we can remove this.

`unmockkAll()` is a mistake. Removed.